egs/aidatatang_200zh/s5/local/nnet3/tuning/run_tdnn_2a.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```

Comment:
please have a look at some other chain/nnet3 scripts and add the performance indicators (dir_info and wers)
Also probably your system should contain only 1 script in tuning/ unless you have two very different architenctures, say tdnn only, tdnn_lstm and tdnn_cnn

Reply:
Thank you, I have added performance indicators in it, and removed /run_tdnn_1a.sh, please check them.